ui/plugins/job_details/main.html

```diff
@@ -7,6 +7,9 @@
       <span ng-if="line.url && line.value.endsWith('raw.log')">
         - <a title="{{line.value}}" href="https://mozilla.github.io/wptview/#/?urls={{line.url | encodeURIComponent}},{{job_details[buildernameIndex].value | encodeURIComponent}}">open in test results viewer</a>
       </span>
+      <span ng-if="line.url && line.value.startsWith('profile_') && line.value.endsWith('.zip')">
+        - <a title="{{line.value}}" href="https://perf-html.io/from-url/{{line.url | encodeURIComponent}}">open in perf-html.io</a>
```

Comment:
These style URLs don't appear to work at the moment? eg:
https://perf-html.io/from-url/https%3A%2F%2Fqueue.taskcluster.net%2Fv1%2Ftask%2FQPUiLGxRRYa_oCnlCpBOzw%2Fruns%2F0%2Fartifacts%2Fpublic%2Ftest_info%2Fprofile_tresize.zip/calltree/?thread&threadOrder=&v=3

Should the URL be passed in the query string instead?

Reply:
The link above doesn't work in either Nightly or Chrome for me:

![screenshot](https://user-images.githubusercontent.com/501702/38311012-f24c63f8-3815-11e8-8533-0fd036dd66e9.png)

Reply:
![screenshot-2018-4-4 perf html](https://user-images.githubusercontent.com/2198968/38311175-897b5ccc-37ec-11e8-8f7f-12c51511aae3.png)

Reply:
I'm on nightly also. Ugh.

Reply:
```bash
$ curl -I "https://perf-html.io/from-url/https%3A%2F%2Fqueue.taskcluster.net%2Fv1%2Ftask %2FQPUiLGxRRYa_oCnlCpBOzw%2Fruns%2F0%2Fartifacts%2Fpublic%2Ftest_info%2Fprofile_tresize.zip/calltree/?thread&threadOrder=&v=3"
HTTP/1.1 404 Not Found
Date: Wed, 04 Apr 2018 13:44:00 GMT
Server: Apache/2.4.29 (Ubuntu)
Strict-Transport-Security: max-age=31536000; includeSubDomains
Content-Type: text/html; charset=iso-8859-1
```

Reply:
I have no idea, I'll have to ask the perf-html.io peeps to have a look.

Reply:
Does the curl command 404 for you too? (I've since edited the comment to correct a typo, so you may need to refresh)

Reply:
And just checking you're using the exact URL above and not one from another job locally?

Reply:
Yes that exact URL - the curl command does 404 but the URL works.

Jmaher just tried it - and it failed at first but worked after he installed the perf-html.io add-on... sorry I should have thought of that, I have the addon also.

Reply:
Instead of 404 it would be much better if perf-html.io could somehow indicate the add-on needs to be installed first. I don't think we should land this, peeps will just get frustrated with a 404 if they don't have the add-on.

Reply:
The add-on shouldn't need to be installed. The URL works for me in Chrome, for example.

The fact that we send a 404 status code is a bug. But the correct page should still show up. So I don't understand why this URL doesn't work for Ed.

Reply:
The server configuration has been fixed and the URL should be working now.